CHANGELOG.md

```diff
@@ -5,6 +5,10 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
+### Added
+- BatchSecretsRetrieval now has an optinal flag allowing the user to specify using the `Accept: base64`
```

Comment:
```suggestion
- BatchSecretsRetrieval now has an optional flag allowing the user to specify using the `Accept: base64`
```

Reply:
actually, this is the wrong changelog message now - rather than adding an optional flag, you added a new method.